materials/terminology.md

```diff
@@ -6,49 +6,32 @@
 While for some there might be differences, the terms "computing cluster", "High Performance Computer (HPC)" and "supercomputer" are also often used interchangeably.
 :::
 
-**Cluster**
-A cluster is all resources wired together for the purpose of high performance computing, which includes computational devices, networking devices (switches) and storage devices combined.
+The term **HPC system** is a stand-alone resource for computationally intensive workloads. 
+They are typically comprised of a multitude of integrated processing and
+storage elements, designed to handle high volumes of data and/or large numbers of floating-point
+operations ([FLOPS](https://en.wikipedia.org/wiki/FLOPS)) with the highest possible performance. 
 
-**Node**
-You can roughly think that one **node** is a single computer.
+To support these constraints, an HPC resource must exist in a specific, fixed location: networking
+cables can only stretch so far, and electrical and optical signals can travel only so fast.
```

Comment:
**Supercomputer** is for computationally intensive workloads with the highest possible performance, designed to handle big volumes of data. It typically includes computational devices, networking devices and storage devices.
To support these constraints, a supercomputer must exist in a specific, fixed location: networking cables can only stretch so far, and electrical and optical signals can travel only so fast.

Reply:
Sorry this went wrong now, but I would replace rows 9-15 and 27 with this.

Reply:
Let me think about this for a moment. Will send another PR with an update regarding this text later. So we can merge this  one for now.